kura/distrib/src/main/resources/common/comment-interfaces-file.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2023 Eurotech and/or its affiliates and others
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#  Eurotech
+#
+import subprocess
+import sys
+import shutil
+from os.path import exists
+
+LOG_MSG_PREFIX = "[comment_interfaces_file.py] "
+ETHERNET_TYPES = ['ethernet', 'eth', 'wired']
+WIRELESS_TYPES = ['wifi', 'wireless']
+INTERFACES_PATH = "/etc/network/interfaces"
+INTERFACES_OLD_PATH =  INTERFACES_PATH + ".old"
+INTERFACES_TMP_PATH = INTERFACES_PATH + ".tmp"
+
+def get_eth_wlan_interfaces_names():
+    """Reads the network interface names using 'nmcli dev' command.
+
+    It is assumed that at least one ethernet interface is present.
+
+    Returns:
+        tuple of lists (eth_names, wlan_names) where:
+            'eth_names' are the found ethernet interface names ordered by name;
+            'wlan_names' are the found wireless interface names ordered by name, might be an empty list.
+    """
+    cmd_output = subprocess.check_output(['nmcli', 'dev']).decode(sys.stdout.encoding).strip()
```

Comment:
This might be a silly comment but: what if, instead of relying on the `nmcli` command, you go ask to NM directly through dbus? `dbusctl` allows for JSON output so that you can parse it directly using the built-in json library and avoid a lot of headaches in parsing the output while being sure to have a reliable API under the hood.

This assumes `dbusctl` is available on the target machine ofc

Reply:
Actually... thinking about it, it would require multiple calls to the DBus interface. Quite more convoluted that what we have here. Ignore my comment.

Reply:
Yeah... I simply adapted the function in `find-net-interfaces.py`...